rllib/execution/parallel_requests.py

```diff
@@ -51,6 +56,10 @@ def asynchronous_parallel_requests(
             (kwargs) as **kwargs to be passed to the `remote_fn`.
             E.g.: actors=[A, B],
             remote_kwargs=[{...} <- **kwargs for A, {...} <- **kwargs for B].
+        return_result_obj_ref_ids: If True, return the object ref IDs of the ready
+        results, otherwise return the actual results.
+        num_requests_to_launch: Number of remote requests to launch on each of the
```

Comment:
Can we describe this more precisely? Like add "if we have not reached `max_remote_requests_in_flight_per_actor` yet, launch exactly `num_requests_to_launch` on the respective actor, regardless of the current values of `remote_requests_in_flight`". Something like this.

Reply:
gotcha -- yeah this is still "rate limited" by the `max_remote_requests_in_flight_per_actor`